app/src/main/java/com/example/filemanager/AllFilesViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.filemanager
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.os.Build
+import android.os.Environment
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import java.io.File
+
+class AllFilesViewModel(
+    @SuppressLint("StaticFieldLeak")
+    private val context: Context
```

Comment:
I don't see where this context is used in the class? Maybe you can remove it ?

Reply:
I deleted context variable